test/com/google/enterprise/adaptor/DocIdSenderTest.java

```diff
@@ -35,6 +30,10 @@
 import java.util.Map;
 import java.util.TreeMap;
 import java.util.concurrent.atomic.AtomicLong;
+import org.junit.Before;
```

Comment:
Can we undo this? It generates a lint error using the old linter, and no one has gotten the new linter working yet.